third_party/move/tools/move-linter/src/model_ast_lints/nonminimal_bool.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+//! This module implements an expression linter that checks for boolean identities
+//! that occur in conditionals:
```

Comment:
```suggestion
```

Reply:
This code checks boolean expressions occurring anywhere, not just in the context of a conditional.

Reply:
Done.